### ISSUE TITLE
a few shortcuts would be nice

### ISSUE THREAD

**Issue by c-cube** [the-lambda-church/coquille#29]
In the spirit of merlin, and other interactive theorem provers, it would be nice to map `\leader<letter>` to some useful commands. For instance, when the cursor is on a symbol, `\leader p` could call `:Coq Print <symbol>` to print the value, and `\leader t` could call `:Coq Check <symbol>`.

A shortcut for SearchAbout would be awesome, too. And syntax coloring in Infos/Goals windows!

**Comment by lgeorget** [comment]
About the syntax coloring in Infos/Goals windows, could you give a try to my fork https://github.com/lgeorget/coquille? Please report bugs. Once I have enough feedback and my coloring is ok, I'll send a pull request here.

**Comment by c-cube** [comment]
It's nice indeed!

**Comment by lgeorget** [comment]
I've submitted the pull requests:
https://github.com/the-lambda-church/coquille/pull/36
https://github.com/the-lambda-church/coquille/pull/35

**Comment by trefis** [comment]
I agree that some shortcuts would be nice, however I don't have the time to implement these functionalities.

That being said, I'd be happy to review and merge if someone sends a pull request for these.
